plugins/modules/vmware_vm_info.py

```diff
@@ -279,6 +279,11 @@ def get_all_virtual_machines(self):
 
             vm_folder = PyVmomi.get_vm_path(content=self.content, vm_name=vm)
             datacenter = get_parent_datacenter(vm)
+            datastore_url = list()
+            for entry in vm.config.datastoreUrl:
+                datastore_url.append({key: gettattr(entry, key)
```

Comment:
Please create a [changelog fragment](https://docs.ansible.com/ansible/latest/community/development_process.html#creating-a-changelog-fragment). See [this fragment](https://github.com/ansible/ansible/blob/c2f528b7686c9573aa8493357219e0bbe4416dd3/changelogs/fragments/reboot-change-default-boot-command.yaml) as an example.

Reply:
Added in changelogs/fragments/datastore_url-added-to-vm_info.yaml. Hope that's fine.